homeassistant/components/decora_wifi/common.py

```diff
@@ -0,0 +1,144 @@
+"""Common code for decora_wifi."""
+
+import logging
+from typing import Dict
+
+# pylint: disable=import-error
```

Comment:
Why is there an import error?

Reply:
The lines in this block are the imports from the external python-decora_wifi API. The previous version of this integration's `light.py` uses it.  Several of the integrations I've looked at suppress the linter error that results from importing from outside homeassistant, but others don't, so maybe that's not the correct thing to do.  Will look into how to fix.

Reply:
Would it possibly have something to do with `decora_wifi` being commented out in `requirements_all.txt`?

`# homeassistant.components.decora_wifi
# decora_wifi==1.4`

Reply:
Re-running `gen_requirements_all`, added the same commented line to `requirements_test_all.txt` and didn't change the one in `requirements_all.txt`, but the commit passed the CI pylint check now so I guess that's sorted.

Reply:
On further research, decora_wifi was added to `COMMENT REQUIREMENTS` in `gen_requirements_all.py` in this PR: https://github.com/home-assistant/core/commit/50c573eb4d3b015ebe40e344b09111d83a0dceeb#diff-ea94dcf9c98793d02e5142e31614ab5c1d1a65c307373ef9706fbfc948180148

Why exactly that was done is a bit above my head. But anyway, due to decora_wifi being commented out in `requirements_all.txt` and `requirements_test_all.txt`, I gather that pylint throws an import error during the test suite because that package isn't automatically loaded into the test environment.

Reply:
Now that workflows are running I can confirm that both pylint and tox are complaining due to not being able to find the upstream API module:

Pylint Error:
```Error: homeassistant/components/decora_wifi/common.py:6:0: E0401: Unable to import 'decora_wifi' (import-error)
Error: homeassistant/components/decora_wifi/common.py:7:0: E0401: Unable to import 'decora_wifi.models.iot_switch' (import-error)
Error: homeassistant/components/decora_wifi/common.py:8:0: E0401: Unable to import 'decora_wifi.models.person' (import-error)
Error: homeassistant/components/decora_wifi/common.py:9:0: E0401: Unable to import 'decora_wifi.models.residence' (import-error)
Error: homeassistant/components/decora_wifi/common.py:10:0: E0401: Unable to import 'decora_wifi.models.residential_account' (import-error)
```

Sample of Failed Tox:
```_________ ERROR collecting tests/components/decora_wifi/test_common.py _________
ImportError while importing test module '/__w/core/core/tests/components/decora_wifi/test_common.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/components/decora_wifi/test_common.py:5: in <module>
    from homeassistant.components.decora_wifi.common import (
homeassistant/components/decora_wifi/__init__.py:17: in <module>
    from .common import (
homeassistant/components/decora_wifi/common.py:6: in <module>
    from decora_wifi import DecoraWiFiSession
E   ModuleNotFoundError: No module named 'decora_wifi'
```

Presumably the pylint check and test suite pass on my development machine because I have the decora_wifi package installed there.

Reply:
Now that the related PR is merged, I'll rebase and adjust this PR to fall in line with that.